bpf/bpf_host.c

```diff
@@ -1022,7 +1022,7 @@ handle_netdev(struct __ctx_buff *ctx, const bool from_host)
 		int ret = DROP_UNSUPPORTED_L2;
 
 		return send_drop_notify(ctx, SECLABEL, WORLD_ID, 0, ret,
-					CTX_ACT_DROP, METRIC_EGRESS);
+					CTX_ACT_DROP, METRIC_INGRESS);
```

Comment:
Is actually depends whether this is called from the `to-netdev` or `to-host` section. We'll need to check `from_host` variable to know.

Reply:
> Is actually depends whether this is called from the `to-netdev` or `to-host` section. We'll need to check `from_host` variable to know.

Oh right. If `from-host` wants `METRIC_EGRESS` then we have it wrong in quite a few more places across `bpf_host` :smile:.